RoboticaUtnFrba/create_autonomy
# Add real IMU

# Description

This PR is part of the [milestone to setup `robot_localization`](https://github.com/RoboticaUtnFrba/create_autonomy/wiki/Tuning-robot-localization) and adds a an IMU to the real robot.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

```bash
# To test in the real robot:
RVIZ=true roslaunch ca_bringup minimal.launch
```

Make sure the ROS topics exist.

**TODO:**

- [x] Check if it's possible to not install `RTIMULib` with `sudo`
- [x] The axis are rotating inverted

**Test Configuration**:

- [X] Real robot
- [ ] Gazebo simulation

# Checklist:

- [x] My code follows the style guidelines of this project (Travis CI is passing)
